gflownet/envs/base.py

```diff
@@ -241,158 +479,38 @@ def reward2proxy(self, reward):
         elif self.reward_func == "identity":
             return self.proxy_factor * reward
         else:
-            raise NotImplemented
-
-    def statetorch2policy(
-        self, states: TensorType["batch", "state_dim"]
-    ) -> TensorType["batch", "policy_input_dim"]:
-        """
-        Prepares a batch of states in torch "GFlowNet format" for the policy
-        """
-        return states
+            raise NotImplementedError
 
-    def state2policy(self, state=None):
-        """
-        Converts a state into a format suitable for a machine learning model, such as a
-        one-hot encoding.
-        """
-        if state is None:
-            state = self.state
-        return state
-
-    def statebatch2policy(self, states: List[List]) -> npt.NDArray[np.float32]:
-        """
-        Converts a batch of states into a format suitable for a machine learning model,
-        such as a one-hot encoding. Returns a numpy array.
-        """
-        return np.array(states)
-
-    def policy2state(self, state_policy: List) -> List:
-        """
-        Converts the model (e.g. one-hot encoding) version of a state given as
-        argument into a state.
-        """
-        return state_policy
-
-    def state2readable(self, state=None):
-        """
-        Converts a state into human-readable representation.
-        """
-        if state is None:
-            state = self.state
-        return str(state)
-
-    def readable2state(self, readable):
-        """
-        Converts a human-readable representation of a state into the standard format.
-        """
-        return readable
-
-    def traj2readable(self, traj=None):
-        """
-        Converts a trajectory into a human-readable string.
-        """
-        return str(traj).replace("(", "[").replace(")", "]").replace(",", "")
-
-    def reset(self, env_id=None):
+    def reset(self, env_id: Union[int, str] = None):
         """
         Resets the environment.
         """
-        self.state = []
+        self.state = self.source.copy()
```

Comment:
I'd like to have it as a method, smth like `self.get_source(reset=True)` to accommodate scenario when we start from random state

Reply:
Yes, to be done when needed :+1: